modules/logging-loki-storage-gcp.adoc

```diff
@@ -1,26 +1,23 @@
 // Module is included in the following assemblies:
-// logging/cluster-logging-loki.adoc
+// logging/log_storage/installing-log-storage.adoc
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-gcp_{context}"]
-= GCP storage
+= Google Cloud Platform storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring Google Cloud Platform storage".

Reply:
Out of scope